htmd/parameterization/dihedral.py

```diff
@@ -54,6 +54,7 @@ def __init__(self):
         self.zeroed_parameters = False
 
         self.parameters = None
+        self.fitdihkeys = None
```

Comment:
`fitdihkeys` -- > `_parameterizable_dihedral_atomtypes`

Reply:
And move it down

Reply:
Renamed and moved